tests/test_hear_server_activate.py

```
@@ -120,5 +122,72 @@ def run_server() -> None:
+	def test_initialization_without_address(self):
+		"""
+		Test multicast server initialization without a valid address.
+
+		Verifies that:
+			1. The logger is initialized with the default name when server_address is None.
+			2. The logger is initialized with the default name when server_address is an empty tuple.
+			3. Cleanup is performed correctly after initialization.
+		"""
+		server = multicast.hear.McastServer(None, None)
+		self.assertIsNotNone(server.logger)
+		self.assertEqual(
```

Ruff (0.15.1)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)